layouts/dashboards/components/header/component.jsx

```diff
@@ -43,21 +44,126 @@ class Header extends PureComponent {
     errorMsg: PropTypes.string,
   };
 
+  renderAreaActions({ isCountryDashboard, isAreaAndCountryDashboard }) {
+    const {
+      downloadLink,
+      locationNames,
+      setAreaOfInterestModalSettings,
+      activeArea,
+    } = this.props;
+
+    const btnTheme = cx(
+      'theme-button-clear theme-button-clear-underline theme-button-small'
+    );
+
+    return (
+      <Dropdown
+        layout="overflow-menu"
+        className="edit-button"
+        onChange={this.handleAreaActions}
+        theme={cx('theme-button-medium theme-dropdown-no-border small square')}
+        options={[
+          {
+            value: 'open_map',
+            component: (
+              <Button
+                theme={btnTheme}
+                link={activeArea && `/map/aoi/${activeArea.id}`}
+              >
+                <Icon icon={dashboardIcon} />
+                Open Map
+              </Button>
+            ),
+          },
+          activeArea &&
+            activeArea.userArea && {
+              value: 'edit_area',
+              component: (
+                <Button
+                  theme={btnTheme}
+                  tooltip={{
+                    text: `Edit ${
+                      locationNames &&
+                      locationNames.adm0 &&
+                      locationNames.adm0.label
+                    }`,
+                    position: 'bottom',
+                  }}
+                  onClick={() => setAreaOfInterestModalSettings({ open: true })}
+                >
+                  <Icon icon={editIcon} />
+                  Edit area
+                </Button>
+              ),
+            },
+          location?.type === 'country' && {
```

Comment:
I think we want the save area button for any dashboard that isnt your saved area.
show if:
- not an area dashboard (i.e. country)
- an area dashboard but doesn't belong to the user already (/aoi/:id in the route but `activeArea.userArea: false`.

Reply:
For example, you should be able to see the 'save' button when viewing one of my areas I have shared with you: https://gfw.global/2PNHTO9